cou/utils/openstack.py

```diff
@@ -54,12 +58,15 @@ def __contains__(self, version: str) -> bool:
         lower_v = Version(self.lower)
         upper_v = Version(self.upper)
         service_version = Version(version)
-        return service_version >= lower_v and service_version < upper_v
+        return lower_v <= service_version < upper_v
+
 
+# pylint: disable=too-few-public-methods
+class OpenStackCodenameLookup:
+    """Class to determine compatible OpenStack codenames for a given component."""
 
-class OpenStackCodenameLookup(object):
     _OPENSTACK_LOOKUP: OrderedDict = OrderedDict()
-    _DEFAULT_CSV_FILE = "cou/utils/openstack_lookup.csv"
+    _DEFAULT_CSV_FILE = os.path.join(os.path.dirname(__file__), "openstack_lookup.csv")
```

Comment:
not-blocking: I suggest to use `Path(__file__).parent` it's more readable.

Reply:
done